openslides/core/views.py

```diff
@@ -24,6 +26,26 @@
 from .serializers import CustomSlideSerializer, TagSerializer
 
 
+class IndexView(utils_views.View):
+    """
+    The primary view for OpenSlides using AngularJS.
+
+    The default base template is 'openslides/core/static/templates/index.html'.
+    You can override it by simply adding a custom 'templates/index.html' file
+    to the custom staticfiles directory. See STATICFILES_DIRS in settings.py.
+    """
+
+    def get(self, *args, **kwargs):
+        """
+        Returns the angular base template.
```

Comment:
This docstring can be removed because the main issue is described in the class docstring.
Or use `Returns an HttpResponse object containing the 'templates/index.html'.`